### PR TITLE
Rollback trivy integration changes back to starboard

### PR DESCRIPTION
**Pull request by KiranBodipi**
1. Reverted changes related to trivy-operator integration - https://github.com/aquasecurity/deployments/pull/522
2. Updated to trivy-operator version to 0.16.1 as the existing image has vulnerabilities.

### CODE REVIEW

File: enforcers/kube_enforcer/kubernetes_and_openshift/manifests/kube_enforcer/003_kube_enforcer_deploy.yaml [hunk 1]
```diff
@@ -93,96 +93,63 @@ spec:
...
     spec:
-      serviceAccountName: trivy-operator
+      serviceAccountName: starboard-operator
       automountServiceAccountToken: true
+      securityContext: {}
       containers:
-        - name: "trivy-operator"
-          image: "docker.io/aquasec/trivy-operator:0.11.0"
+        - name: operator
+          image: docker.io/aquasec/starboard-operator:0.15.15
```
[the remark this answers is not in the document]
Comment (KiranBodipi) [reply enforcers/kube_enforcer/kubernetes_and_openshift/manifests/kube_enforcer/003_kube_enforcer_deploy.yaml:117]: > Is it possible to utilise the most recent release?

Lets not change it for now, as it requires testing effort for both manifests and also aqua-helm.

**Review by deven0t — APPROVED** [review]
LGTM!!!

**Comment by KiranBodipi** [comment]
Hey @semyonmor @KoppulaRajender can you help to review and merge this PR

**Review by mjshastha — APPROVED** [review]
I'm content with the changes.

**Review by KoppulaRajender — APPROVED** [review]

**Pull request merged**